pkg/armclient/types.go

```diff
@@ -285,3 +285,78 @@ type ActivityLogResource struct {
 		} `json:"httpRequest,omitempty"`
 	} `json:"value"`
 }
+
+// ContainerLogResponse for container logs
+type ContainerLogResponse struct {
+	Content string `json:"content"`
+}
+
+// ContainerGroupResponse is the response to a get request on a container group
+type ContainerGroupResponse struct {
```

Comment:
Do we want to put these types in here? Could become quite large as we add more data plane functionality.

Reply:
Yeah I agree, I'll track an issue for re-orging this stuff in a separate PR.